packages/cli-kit/src/store.ts

```diff
@@ -40,17 +41,66 @@ const schema = {
 
 let _instance: CLIKitStore | undefined
 
-export function cliKitStore() {
+export async function cliKitStore() {
```

Comment:
I'm going to go ahead and assume you've vetted carefully that we're updating every place we're calling this function to ensure we're awaiting as appropriate.

Reply:
Yep 👍